examples/hosts/app-integration/external-data/src/start.ts

```diff
@@ -12,16 +12,16 @@ import type { IAppModel } from "./modelInterfaces";
 import { DebugView, AppView } from "./view";
 import { TaskListContainerRuntimeFactory } from "./model";
 
-const updateTabForId = (id: string) => {
+const updateTabForId = (id: string): void => {
     // Update the URL with the actual ID
     location.hash = id;
 
     // Put the ID in the tab title
     document.title = id;
 };
 
-const render = (model: IAppModel) => {
-    const appDiv = document.getElementById("app") as HTMLDivElement;
+const render = (model: IAppModel): void => {
+    const appDiv = document.querySelector("#app") as HTMLDivElement;
```

Comment:
Is this actually preferred now?  I feel like getElementById at least used to be preferred in this usage for slightly better performance (though it doesn't really matter in this case).

Reply:
This is coming from the `unicorn` ruleset we extend. I don't have strong opinions one way or the other, but in theory this at least helps enforce consistency.

Reply:
Ok, no strong preference from me.

Reply:
[unicorn docs](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-query-selector.md) The rationale seems to be twofold: consistency and having to know fewer APIs.